custom_components/yamaha_ynca/services.py

```
@@ -18,18 +25,24 @@
+    if config_entry is None or config_entry.state is not ConfigEntryState.LOADED:
+        raise ServiceValidationError(
+            translation_domain=DOMAIN,
+            translation_key="config_entry_not_found",
+            translation_placeholders={
+                "config_entry_id": call.data.get(ATTR_CONFIG_ENTRY_ID)
+            },
+        )
+
+    # Handle actual call
+    for line in call.data.get(ATTR_RAW_DATA, "").splitlines():
+        line = line.strip()  # noqa: PLW2901
```

Ruff (0.14.7)

Unused `noqa` directive (non-enabled: `PLW2901`)

Remove unused `noqa` directive

(RUF100)